mmaction/core/evaluation/eval_hooks.py

```diff
@@ -187,3 +187,27 @@ def after_train_epoch(self, runner):
                 self.best_json['best_ckpt'] = current_ckpt_path
                 self.best_json['key_indicator'] = self.key_indicator
                 mmcv.dump(self.best_json, json_path)
+
+class EvalPlusBeforeRunHook(EvalHook):
+    """Evaluation hook, adds evaluation before training.
+    """
+    def before_run(self, runner):
+        from mmaction.apis import single_gpu_test
+        results = single_gpu_test(runner.model, self.dataloader, show=False)
+        self.evaluate(runner, results)
+
+
+class DistEvalPlusBeforeRunHook(DistEvalHook):
+    """Distributed evaluation hook, adds evaluation before training.
+    """
+
+    def before_run(self, runner):
+        from mmaction.apis import multi_gpu_test
+        results = multi_gpu_test(
+            runner.model,
+            self.dataloader,
+            tmpdir=osp.join(runner.work_dir, '.eval_hook'),
+            gpu_collect=self.gpu_collect)
+        if runner.rank == 0:
+            print('\n')
```

Comment:
why `print`?

Reply:
Part of [mmdetection eval_hooks](https://github.com/open-mmlab/mmdetection/blob/682f03db0c0516324ec4e1e5d97e1d93c0a2485a/mmdet/core/evaluation/eval_hooks.py#L55), which this code is heavily based off of for NNCF implementation.

Reply:
ok, if it's a part of global formatting during eval